### PR TITLE
[Southpaw] - sage - Create creating-your-first-smart-contract-on-the-celo-blockchain

### PR DESCRIPTION
**Pull request by Southpaw0**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

**Review by Ishanpathak1 — COMMENTED** [review]
Metadata is missing and ,have 1 line space after every heading

File: blog/creating-your-first-smart-contract-on-the-celo-blockchain/creating-your-first-smart-contract-on-the-celo-blockchain.md [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+      address[] public contributors; 
+}
+
+```
+
+In our contract, we have a variable `owner` of type `address`, which we will use to store the `address` of the contract creator/owner, we also created a `minimumContibution` variable of type `uint` that would receive the value for the minimum amount of contribution that can be made to the contract and also a list of `address` to store contributors who have donated to the smart contract.
+
+Next, we set up our contract’s constructor, which will require the value of the minimum amount a contributor can make to the smart contract, as its input.
+
+```Solidity
```
Comment (viral-sangani) [inline blog/creating-your-first-smart-contract-on-the-celo-blockchain/creating-your-first-smart-contract-on-the-celo-blockchain.md:78]:
```suggestion
```solidity
```

File: blog/creating-your-first-smart-contract-on-the-celo-blockchain/creating-your-first-smart-contract-on-the-celo-blockchain.md [hunk 2]
```diff
@@ -0,0 +1,291 @@
+---
+title: How to create your first smart contract on the celo blockchain,
+description: This tutorials teaches you how to create your first sample smart contract and deploy it on the celo blockchain,
+
+authors:
+  - name: Israel Okunaya
+    title: Software Engineer, Technical writer @Celo Foundation
+    url: https://github.com/Southpaw0
+    image_url:
```
Comment (Ishanpathak1) [inline blog/creating-your-first-smart-contract-on-the-celo-blockchain/creating-your-first-smart-contract-on-the-celo-blockchain.md:9]: Image_url can't be empty= You can take that from your github

**Review by Ishanpathak1 — COMMENTED** [review]
Make the changes

**Review by viral-sangani — APPROVED** [review]

**Pull request merged**